xbmc/guilib/GUIFontTTF.cpp

```diff
@@ -469,7 +469,11 @@ void CGUIFontTTF::DrawTextInternal(CGraphicContext& context,
         break;
       cursorX += ch->m_advance;
     }
+
+    // Reserve vertices space in vector: 4 * num glyphs
+    tempVertices->reserve(glyphs.size() * 4);
```

Comment:
This is not essential. It's just to reduce number of memory re-allocations in large texts.